tests/test_main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+from app.main import is_isogram
+
+
+def test_empty_string():
+    assert is_isogram("") is True
+
+
+def test_case_insensitive():
+    assert is_isogram("LKJFSDLIlsdfeojf") is False
```

Comment:
Bad test because in `LKJFSDLIlsdfeojf` you have repeating both sensitive and insensitive letters. Make two different test to test `test_repeated_insensitive` and `test_repeated_sensitive`.